server/routes/induction/update/previousWorkExperienceTypesUpdateController.ts

```diff
@@ -89,38 +94,30 @@ export default class PreviousWorkExperienceTypesUpdateController extends Previou
       `Previous Work Experiences changes resulting in going to the Detail pages for ${workExperienceTypesToShowDetailsFormFor}`,
     )
 
-    // TODO - replace with correct redirect once we've implemented form redirect/handling for a "queue" of Previous Work Experience Detail pages
-    return res.redirect(`/plan/${prisonNumber}/view/work-and-interests`)
-  }
+    // Update the InductionDTO in the session with changes to Previous Work Experiences, but do not persist to the API
+    // The user will be redirected to each Previous Work Experience Detail page in turn
+    req.session.inductionDto = updatedInductionDtoWithPreviousWorkExperiences(
```

Comment:
This updates the Induction DTO on the session by calling the `updatedInductionDtoWithPreviousWorkExperiences` method, which adds in any new work experience types.

At the moment we are only redirecting to the first new job type detail page, but I will change that in my next PR (to iterate through them all).
I just wanted to get this first part done to prove whether it will work or not ... and I think it does 👍 😁 🎉